scripts/license/check.py

```diff
@@ -16,31 +16,30 @@
 Checks whether files have an appropriate license header.
 """
 
-import os
 from pathlib import Path
 import subprocess
 
 from license_header import license_header, has_license_header
 
-PROJECT_ROOT = subprocess.check_output(
-    "git rev-parse --show-toplevel".split()).strip().decode("utf-8")
-target_dirs = [os.path.join(PROJECT_ROOT, d) for d in os.listdir(PROJECT_ROOT)
-               if d != "vendor" and os.path.isdir(os.path.join(PROJECT_ROOT, d))]
+PROJECT_ROOT = Path(subprocess.check_output(
+    "git rev-parse --show-toplevel".split()).strip().decode("utf-8"))
 
 
 def main(verbose=False):
     ok = True
 
-    ok &= check(Path(PROJECT_ROOT).glob("*.go"), license_header("//"), verbose)
-    ok &= check(Path(PROJECT_ROOT).glob("*.py"), license_header("#"), verbose)
-    ok &= check(Path(PROJECT_ROOT).glob("*.sh"), license_header("#"), verbose)
+    ok &= check(PROJECT_ROOT.glob("*.go"), license_header("//"), verbose)
```

Comment:
We can move this into the loop below, but maybe the current code is easier to read?

Reply:
Either is fine, but I personally prefer the current code.